src/main/java/org/spongepowered/api/item/merchant/TradeOffer.java

```diff
@@ -38,6 +38,13 @@
 public interface TradeOffer {
 
     /**
+     * Get the {@link TradeOfferBuilder} used to edit trade offers.
+     *
+     * @return The trade offer builder
+     */
+    TradeOfferBuilder builder();
```

Comment:
This doesn't belong here - there are already methods for getting builders in `GameRegistry`.